### PR TITLE
PMM-13044 nightly and aws staging with new pmm framework

### PR DESCRIPTION
**Pull request by yurkovychv**

### CODE REVIEW

File: pmm/v3/pmm3-aws-staging-start.groovy [hunk 1]
```diff
@@ -34,24 +34,14 @@ pipeline {
             name: 'SSH_KEY'
         )
         string(
-            defaultValue: 'pmm2023fortesting!',
+            defaultValue: 'pmm3admin',
```
Comment (puneet0191) [inline pmm/v3/pmm3-aws-staging-start.groovy:37]: could you make it `pmm3admin!`

File: pmm/v3/pmm3-aws-staging-start.groovy [hunk 2]
```diff
@@ -352,42 +272,40 @@ pipeline {
...
+                        docker network create pmm-qa || true
 
-                        sudo mkdir -p /srv/pmm-qa || :
-                        pushd /srv/pmm-qa
-                            sudo git clone --single-branch --branch ${PMM_QA_GIT_BRANCH} --depth=1 https://github.com/percona/pmm-qa.git .
+                        sudo mkdir -p /srv/qa-integration || :
+                        pushd /srv/qa-integration
+                            sudo git clone --single-branch --branch ${PMM_QA_GIT_BRANCH} https://github.com/Percona-Lab/qa-integration.git .
                             sudo git checkout ${PMM_QA_GIT_COMMIT_HASH}
                             sudo curl -O https://raw.githubusercontent.com/Percona-QA/percona-qa/master/get_download_link.sh
```
Comment (puneet0191) [inline pmm/v3/pmm3-aws-staging-start.groovy:285]: we don't need this anymore? @saikumar-vs

**Review by puneet0191 — APPROVED** [review]
Just left a few comments

File: pmm/v3/pmm3-aws-staging-start.groovy [hunk 3]
```diff
@@ -63,83 +53,57 @@ pipeline {
...
+                QUERY_SOURCE=perfschema|slowlog
+                SETUP_TYPE=replica(Replication)|gr(Group Replication)|(single node if no option passed)
+            --database mysql - Official MySQL (ex: --database mysql,QUERY_SOURCE=perfschema)
+            Additional options:
+                QUERY_SOURCE=perfschema|slowlog
+            --database psmdb - Percona Server for MongoDB (ex: --database psmdb=latest,SETUP_TYPE=pss)
+            Additional options:
+                SETUP_TYPE=pss(Primary-Secondary-Secondary)|psa(Primary-Secondary-Arbiter)|shards(Sharded cluster)
+            --database pdpgsql - Percona Distribution for PostgreSQL (ex: --database pdpgsql=16)
+            --database pgsql - Percona Distribution for PostgreSQL (ex: --database pgsql=16)
```
Comment (saikumar-vs) [inline pmm/v3/pmm3-aws-staging-start.groovy:103]: Please change pgsql to Official PostgreSQL Distribution

File: pmm/v3/pmm3-ui-tests-nightly.groovy [hunk 4]
```diff
@@ -266,45 +210,37 @@ pipeline {
...
-             when {
-                expression { env.GIT_COMMIT_HASH.length()>0 }
-            }
-            steps {
-                sh 'git checkout ' + env.GIT_COMMIT_HASH
-            }
-        }
         stage('Start Server') {
             steps {
-                runStagingServer(DOCKER_VERSION, CLIENT_VERSION, '--addclient=haproxy,1 --setup-alertmanager --setup-external-service', CLIENT_INSTANCE, '127.0.0.1', ADMIN_PASSWORD)
```
Comment (saikumar-vs) [inline pmm/v3/pmm3-ui-tests-nightly.groovy:285]: Maybe its somewhere but i couldnt locate haproxy setup?

**Review by saikumar-vs — CHANGES_REQUESTED** [review]
Thank you for working on these changes, the looks nice, couple of comments pls have a look.

**Review by saikumar-vs — APPROVED** [review]

**Pull request merged**